R/rules-indention.R

```diff
@@ -54,12 +54,21 @@ indent_op <- function(pd,
 indent_eq_sub <- function(pd,
                           indent_by,
                           token = c("EQ_SUB", "EQ_FORMALS")) {
-  eq_sub <- which(pd$token %in% token)
```

Comment:
@MichaelChirico changing to boolean here because I think it's slightly more efficient, but later realised we need to convert to int again for handling your exception

Reply:
Thanks for fixing. How did you know to look in `indent_eq_sub` if I might ask?

Reply:
Oh I just saw your comment on the issue, nvm.

Reply:
I just full text searched the rules in `R/rules-indention.R` With “eq_sub” to find a rule that has this in the name.